homeassistant/components/sql/sensor.py

```diff
@@ -56,10 +60,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         sess.execute("SELECT 1;")
 
     except sqlalchemy.exc.SQLAlchemyError as err:
-        _LOGGER.error("Couldn't connect using %s DB_URL: %s", db_url, err)
+        _LOGGER.error(
+            "Couldn't connect using %s DB_URL: %s",
+            DB_URL_RE.sub("//", str(db_url)),
+            DB_URL_RE.sub("//", str(err)),
```

Comment:
We should replace it with `***`, as it now isn't clear anymore if there was a username/password or not.

We could also use our `HideSensitiveDataFilter` on this logger alternatively.